client/lib/invites/reducers/invites-create-validation.js

```diff
@@ -8,6 +8,16 @@ import { fromJS } from 'immutable';
  */
 import { action as ActionTypes } from 'lib/invites/constants';
 
+function normalizeSuccessValidations( validations ) {
```

Comment:
I think we could move it to where `RECEIVE_CREATE_INVITE_VALIDATION_SUCCESS` is dispatched.